x/fundraising/keeper/keeper.go

```diff
@@ -99,28 +98,38 @@ func (k Keeper) SetParams(ctx sdk.Context, params types.Params) {
 	k.paramSpace.SetParamSet(ctx, &params)
 }
 
-// ReserveCreationFee reserves the auction creation fee to the fee collector account.
-func (k Keeper) ReserveCreationFee(ctx sdk.Context, auctioneerAddr sdk.AccAddress) error {
+// PayCreationFee reserves the auction creation fee to the fee collector account.
```

Comment:
```suggestion
// PayCreationFee sends the auction creation fee to the fee collector account.
```

Reply:
I forgot to update the comment. Thank you for pointing it out!